### PR TITLE
fix: package google fonts and do not render fallback fonts

### PR DESCRIPTION
**Pull request by jhasuraj01**
References: -
- https://developers.google.com/fonts/docs/getting_started#use_font-display
- https://www.w3.org/TR/css-fonts-4/#font-display-desc

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6272 On cysync the font is getting downloaded after opening](https://app.clickup.com/t/9002019994/PRF-6272)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6272 On cysync the font is getting downloaded after opening](https://app.clickup.com/t/9002019994/PRF-6272)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6272 On cysync the font is getting downloaded after opening](https://app.clickup.com/t/9002019994/PRF-6272)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6272 On cysync the font is getting downloaded after opening](https://app.clickup.com/t/9002019994/PRF-6272)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6272 On cysync the font is getting downloaded after opening](https://app.clickup.com/t/9002019994/PRF-6272)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6272 On cysync the font is getting downloaded after opening](https://app.clickup.com/t/9002019994/PRF-6272)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6272 On cysync the font is getting downloaded after opening](https://app.clickup.com/t/9002019994/PRF-6272)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6272 On cysync the font is getting downloaded after opening](https://app.clickup.com/t/9002019994/PRF-6272)

File: apps/desktop/html/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
-      href="https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@300;400;500;700&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      href="https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@300;400;500;700&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=block"
```
Comment (irshadCypherock) [inline apps/desktop/html/index.html]: It'll be better if we can package the font with the application

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**